src/openrct2/world/park.c

```diff
@@ -111,8 +111,6 @@ void park_init()
     reset_researched_scenery_items();
 
     gParkEntranceFee = MONEY(10, 00);
-    gPeepSpawns[0].x = PEEP_SPAWN_UNDEFINED;
-    gPeepSpawns[1].x = PEEP_SPAWN_UNDEFINED;
```

Comment:
The only time `park_init()` is called is during `game_init_all(sint32 mapsize)`, which also calls `reset_park_entry()`. `reset_park_entry()` sets all peep spawns to null, so there is no sense in having the first 2 peep spawns reset, only to later have all peep spawns reset in the same function.

I meant to make a seperate branch for this but I have not learned yet how to make a branch of only a few lines of code from a file. (if it's even possible).